workers/unity/Assets/Fps/Scripts/GameLogic/StateMachine/Session/GetDeploymentsState.cs

```diff
@@ -41,7 +41,15 @@ public override void Tick()
                 return;
             }
 
-            if (!result.HasValue || result.Value.Status.Code != ConnectionStatusCode.Success)
+            if (!result.HasValue)
+            {
+                Manager.ScreenManager.StartScreenManager.ShowFailedToGetDeploymentsText(
+                    $"Failed to retrieve any login tokens.\n Unknown error.");
```

Comment:
Unknown error. More like - please report this to us as this should literally never happen 😂